plugins/modules/helm.py

```diff
@@ -78,6 +78,13 @@
     default: {}
     aliases: [ values ]
     type: dict
+  values_files:
+    description:
+        - Value files to pass to chart.
```

Comment:
Should we note that these paths will be read from the target host's filesystem, rather than the one that is running Ansible?

Reply:
Also maybe note that order matters, and clarify what the precedence will be if I supply both `values` and `values_files`

Reply:
Where would you like me to add this information?

Reply:
Just in the parameter descriptions here, I think that's usually where this sort of info goes

Reply:
added